### PR TITLE
Allow the AI to unbolt their core, added a noise for it too.

### PR DESCRIPTION
**Pull request by lbnesquik**
## About The Pull Request
This is a simple which allows AI's to unbolt. There is little usage and it will probably get you killed, but hey, the option is here.

## Changelog
:cl:
expansion: Allow the AI to unbolt their core
/:cl:

### CODE REVIEW

**Comment by neohenry** [comment]
The only time AI will unbolt is when xenomorphs hijack and the AI will be unsafe if remained in the AI core.

File: code/modules/mob/living/silicon/ai/ai_verbs.dm [hunk 1]
```diff
@@ -226,6 +226,21 @@
 	popup.set_content(dat)
 	popup.open(FALSE)
 
+/mob/living/silicon/ai/verb/toggle_anchor()
```
Comment (BraveMole) [inline code/modules/mob/living/silicon/ai/ai_verbs.dm:229]: Don't add new verbs.

Make this an ability with a proper keybind, or something like that

**Review by BraveMole — CHANGES_REQUESTED** [review]

**Comment by lbnesquik** [comment]
There was a discussion in dev general where that verb usage was said to be okay.

**Comment by Ieteihowmee** [comment]
Many times this would have been handy

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**